core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -541,13 +541,10 @@ public void start(Boolean isPrimary) throws IOException {
               new LostFileDetector(this, mInodeTree),
               (int) Configuration.getMs(PropertyKey.MASTER_HEARTBEAT_INTERVAL_MS)));
       if (Configuration.getBoolean(PropertyKey.MASTER_STARTUP_CONSISTENCY_CHECK_ENABLED)) {
-        mStartupConsistencyCheck = getExecutorService().submit(new Callable<List<AlluxioURI>>() {
-          @Override
-          public List<AlluxioURI> call() throws Exception {
-            return startupCheckConsistency(ExecutorServiceFactories
-                .fixedThreadPoolExecutorServiceFactory("startup-consistency-check", 32).create());
-          }
-        });
+        mStartupConsistencyCheck = getExecutorService().submit(() -> startupCheckConsistency(
+            ExecutorServiceFactories
+               .fixedThreadPoolExecutorServiceFactory("startup-consistency-check", 32).create()));
+      }
```

Comment:
failed to build: `DefaultFileSystemMaster.java:557:10: unexpected token: void`
do you have all parentheses closed?